### ISSUE TITLE
containerd package has the same tarball checksum problems as others

### ISSUE THREAD

**Issue by afbjorklund** [kubernetes/minikube#13193]
Building the ISO fails, since the upstream git archive randomly changes checksum.

This is due to including `vendor/k8s.io/client-go/pkg/version/.gitattributes`

```
base.go export-subst
```

The substitution will corrupt the git archive, by filling the git info with wrong information.

```diff
58,59c58,59
< 	gitVersion   string = "v0.0.0-master+$Format:%h$"
< 	gitCommit    string = "$Format:%H$" // sha1 from git, output of $(git rev-parse HEAD)
---
> 	gitVersion   string = "v0.0.0-master+1e5ef943eb"
> 	gitCommit    string = "1e5ef943eb76627a6d3b6de8cd1ef6537f393a71" // sha1 from git, output of $(git rev-parse HEAD)
```

It was intended to record the kubernetes version, but here has a random short rev of containerd.

Problems started with containerd 1.5.x, that started including `k8s.io/client-go` (unlike 1.4.x)

**Comment by afbjorklund** [comment]
Broke the ISO build in #13089

**Comment by afbjorklund** [comment]
Reported as https://github.com/containerd/containerd/issues/6387

**Comment by sharifelgamal** [comment]
Yeah, we've seen this happen on occasion for containerd and crio, not a whole lot we can do about it except fix the hash for the time being.

**Comment by afbjorklund** [comment]
Fixed in containerd 1.6
